### PR TITLE
Upgrade OpenJDK to 11

### PR DESCRIPTION
**Pull request by a1exsh**
Staying on top of the change.

Internal ticket: 858

### CODE REVIEW

**Comment by a1exsh** [comment]
:+1:

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/library/openjdk-8:latest
+FROM registry.opensource.zalan.do/library/openjdk-11-jre-slim:latest
```
Comment (antban) [inline Dockerfile:1]: Not sure, but maybe `openjdk-11-jdk-slim` is the better option (we anyways won't save a lot of time with such a big installation steps afterwards. 
Plus kafka instances are not recreated every now and then. 
But the jdk tooling may be very useful on kafka instace...

Comment (a1exsh) [reply Dockerfile:1]: Good point.  I was looking at our nakadi base image, but considerations are very different there.

**Comment by a1exsh** [comment]
:+1:

**Comment by antban** [comment]
:+

**Comment by antban** [comment]
:+1:

**Pull request merged**